_posts/2018-01-11-uber.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Uber
+sourceURL: https://eng.uber.com/m-uber/
```

Comment:
Nit. Remove trailing slash?

Reply:
No. If you remove the trailing slash, the browser will request it without the trailing slash and then get 302 redirected to the version with the slash. On a mobile device on a slow connection, that redirect can be a couple hundred milliseconds or more.

If the URL were being displayed, then yes, it might make sense to remove the trailing slash. But in a case like this, adding the trailing slash is a tiny bit of time-savings to people who follow the link.